src/components/Header/Notifications.js

```diff
@@ -12,7 +13,9 @@ const Notifications = ({ plan, notifications, emptyNotificationText }) => {
       <div className="user-menu helper--right dp-notifications">
         <div className="dp-msj-notif">
           {!showNotifications ? (
-            <i>{emptyNotificationText}</i>
+            <i>
+              <FormattedMessage id="empty_notification_text" />
+            </i>
```

Comment:
```js
            <i>{emptyNotificationText ||  <FormattedMessage id="empty_notification_text" />}</i>
```

```js
            <i>{emptyNotificationText ? emptyNotificationText 
                :  <FormattedMessage id="empty_notification_text" />}</i>
```

Reply:
vale la pena que tomemos el texto de empty desde el BE? no es mejor tomarlo de los archivos de recursos?

Reply:
La ventaja de intentar tomarlo desde el backend es que por defecto vamos a tener consistencia con el comportamiento actual de Doppler, pero agregamos un poquito de complejidad.

No tengo una posición fuerte en este tema.